scripts/debug-env-loading.js

```
@@ -0,0 +1,140 @@
+// Check different loading methods
+console.log('\n1. CHECKING RAW ENVIRONMENT VARIABLES:');
+console.log('-----------------------------------------');
+console.log('QSTASH_TOKEN exists:', !!process.env.QSTASH_TOKEN);
+console.log('QSTASH_CURRENT_SIGNING_KEY exists:', !!process.env.QSTASH_CURRENT_SIGNING_KEY);
+console.log('QSTASH_NEXT_SIGNING_KEY exists:', !!process.env.QSTASH_NEXT_SIGNING_KEY);
+
+// Try loading with dotenv
+console.log('\n2. LOADING WITH DOTENV:');
+console.log('-----------------------------------------');
+try {
+  const dotenv = require('dotenv');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)